src/lib/ilqp.js

```diff
@@ -56,26 +25,21 @@ function _serializeQuoteRequest (requestParams) {
 /**
   * @param {Object} params
   * @param {Object} params.plugin The LedgerPlugin used to send quote request
-  * @param {String} params.connector The ILP address of the connector to quote from
   * @param {Object} params.quoteQuery ILQP request packet parameters
   * @param {Integer} [params.timeout] Milliseconds
   * @returns {Object} Ilqp{Liquidity,BySourceAmount,ByDestinationAmount}Response or IlpError
   */
 function quoteByConnector ({
   plugin,
-  connector,
   quoteQuery,
   timeout
 }) {
-  const prefix = plugin.getInfo().prefix
+  plugin = compat(plugin)
   const requestPacket = _serializeQuoteRequest(quoteQuery)
   const requestType = requestPacket[0]
 
-  debug('remote quote connector=' + connector, 'query=' + JSON.stringify(quoteQuery))
+  debug('remote quote', 'query=' + JSON.stringify(quoteQuery))
   return plugin.sendRequest({
```

Comment:
Same question as https://github.com/interledgerjs/ilp-connector/pull/401#discussion_r157580607

Reply:
So then is this PR intended to preserve or break support for ILQP?

Reply:
Preserve support for the moment, even though it's a deprecated feature

Reply:
I think it would be more precise to say that we are experimenting with the idea of deprecating ILQP, not that we decided to deprecate it?

Reply:
Maybe, but if someone were going to write a new implementation of ILP right now I wouldn't recommend implementing ILQP